libs/server/IncrErrorType.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Garnet.server
+{
+    /// <summary>
+    /// Type of error when executing Increment/Decrement commands
+    /// </summary>
+    public enum IncrErrorType : byte
```

Comment:
>IncrErrorType [](http://example.com/codeflow?start=16&length=13)

May be this could be renamed to be a generic OperationErrorType that could be potentially reused for other operation errors.